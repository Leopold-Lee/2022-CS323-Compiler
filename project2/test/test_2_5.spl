float one() {
  return 0.1 + 0.1;
}

float add(float x,float y){
  return x+y+one();
}
int foo()
{
  float a = 0.5;
  float b = 0.5;
  int k;
  k = add(a,b);
  return a;
}