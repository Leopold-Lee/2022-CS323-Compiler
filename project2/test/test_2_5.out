Error type 5 at Line 13: unmatching type on both sides of assignment
Error type 8 at Line 14: incompatiable return type
